docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -520,12 +521,13 @@ When the application starts, the TLS registry performs some checks to ensure the
 
 If any of these checks fail, the application will fail to start.
 
+[#reloading-certificates]
 == Reloading Certificates
 
 The `TlsConfiguration` obtained from the `TLSConfigurationRegistry` includes a mechanism for reloading certificates.
 The `reload` method refreshes the key stores and trust stores, typically by reloading them from the file system.
 
-NOTE: The reload operation is not automatic and must be triggered manually. Additionally, the `TlsConfiguration` implementation must support reloading (which is the case for the configured certificate).
+NOTE: The reload operation is not automatic and must be triggered manually.Additionally, the `TlsConfiguration` implementation must support reloading (which is the case for the configured certificate).
```

Comment:
You need to revert this change.

Reply:
That's me hitting a key to shutdown my screen saver...